guide/src/class.md

```diff
@@ -181,6 +182,59 @@ created from Rust, but not from Python.
 
 For arguments, see the [`Method arguments`](#method-arguments) section below.
 
+## Initializer
+
+An initializer implements Python's `__init__` method.
+
+It may be required when it's needed to control an object initalization flow on the Rust code.
+For example, you define a class that extends `PyDict` and don't want that the original
+`__init__` method of `PyDict` been called. In this case by defining an own `init` method
+it's possible to stop initialization flow.
+
+If you declare an own `init` method you may need to call a super class `__init__` method
+explicitly like that happens in a regular Python code.
+
+To declare an initializer, you need to define a method and annotate it with the `#[init]`
+attribute. An `init` method must have the same input paretemeters signature like
+in the constructor method.
+
+Like in the constructor case the Rust method name isn't important.
+
+```rust,ignore
```

Comment:
Why is this ignored? Please write some test code in this example. It can be something simple like a hidden test that just creates and checks it.